src/FSharpPlus/Builders.fs

```diff
@@ -119,7 +119,7 @@ module Builders =
             fix ()
         member inline this.For (p: #seq<'T>, rest: 'T->'``MonadPlus<'U>``) : '``MonadPlus<'U>`` =
             let mutable isReallyDelayed = true
-            Delay.Invoke (fun () -> isReallyDelayed <- false; Unchecked.defaultof<'``MonadPlus<'U>``>) |> ignore
+            Delay.Invoke (fun () -> isReallyDelayed <- false; Empty.Invoke () : '``MonadPlus<'U>``) |> ignore
```

Comment:
:+1:

Reply:
Note that there are no additional static members AFAIK.
Empty should be already there as it's a monadplus builder.

In the other case, return should be also there as it is a monad.